Apps/WebClient/src/Server/Controllers/UserProfileController.cs

```diff
@@ -303,6 +331,64 @@ public async Task<IActionResult> GetUserEmailInvite(string hdid)
             }
         }
 
+        /// <summary>
```

Comment:
I don't understand the purpose of this method.  The actual validation code for SMS should  never be returned to the client and the method above does validation.  Can you elaborate?

Reply:
its only used to check if the phone was validated or not, this is the part that I mention that could use some refactoring (together with email validation) the actual code/digit is not sent to the front end.